src/components/ContentHighlights/data/constants.js

```diff
@@ -23,19 +37,54 @@ export const MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET = 12;
 export const HIGHLIGHT_TITLE_MAX_LENGTH = 60;
 // Stepper Step Text that match testing components
 export const STEPPER_STEP_TEXT = {
-  createTitle: 'Create a title for your highlight',
-  selectContent: 'Add content to your highlight',
-  confirmContent: 'Confirm your selections',
-  getConfirmContentSubtitle: (highlightTitle) => `Review content selections for "${highlightTitle}"`,
+  HEADER_TEXT: {
+    createTitle: 'Create a title for your highlight',
+    selectContent: 'Add content to your highlight',
+    confirmContent: 'Confirm your selections',
+  },
+  SUB_TEXT: {
+    createTitle: `Create a unique title for your highlight. This title is visible
+                  to your learners and helps them navigate to relevant content.`,
+    selectContent: (highlightTitle) => sanitizeAndParseHTML(`Select up to <strong>${MAX_CONTENT_ITEMS_PER_HIGHLIGHT_SET}</strong> items for "${highlightTitle}".
```

Comment:
Went with this implementation of parsing, sanitized strings, as opposed to the internalization `formatMessage` as we discussed.
https://formatjs.io/docs/react-intl/upgrade-guide-4x/#migrating-off-embedded-html-in-messages

Will consider internationalization as tech debt for the future, and used this implementation for the turnaround time, and cleaner implementation.